src/effects/backends/builtin/pitchshifteffect.cpp

```diff
@@ -0,0 +1,98 @@
+#include "effects/backends/builtin/pitchshifteffect.h"
+
+#include "util/sample.h"
+
+PitchShiftGroupState::~PitchShiftGroupState() noexcept {
+    SampleUtil::free(m_retrieveBuffer[0]);
+    SampleUtil::free(m_retrieveBuffer[1]);
+
+    VERIFY_OR_DEBUG_ASSERT(m_pRubberBand) {
+        return;
+    }
+
+    m_pRubberBand->reset();
+}
+
+// static
+QString PitchShiftEffect::getId() {
+    return "org.mixxx.effects.pitchshift";
+}
+
+//static
+EffectManifestPointer PitchShiftEffect::getManifest() {
+    EffectManifestPointer pManifest(new EffectManifest());
+
+    pManifest->setId(getId());
+    pManifest->setName(QObject::tr("Pitch Shift"));
+    pManifest->setShortName(QObject::tr("Pitch Shift"));
+    pManifest->setAuthor("The Mixxx Team");
+    pManifest->setVersion("1.0");
+    pManifest->setDescription(QObject::tr(
+            "Raises or lowers the original pitch of a sound."));
+
+    EffectManifestParameterPointer pitch = pManifest->addParameter();
+    pitch->setId("pitch");
+    pitch->setName(QObject::tr("Pitch"));
+    pitch->setShortName(QObject::tr("Pitch"));
+    pitch->setDescription(QObject::tr(
+            "The new pitch of a sound."));
+    pitch->setValueScaler(EffectManifestParameter::ValueScaler::Linear);
+    pitch->setUnitsHint(EffectManifestParameter::UnitsHint::Unknown);
+    pitch->setDefaultLinkType(EffectManifestParameter::LinkType::Linked);
+    pitch->setDefaultLinkInversion(EffectManifestParameter::LinkInversion::Inverted);
```

Comment:
why inverted?
```suggestion
    pitch->setDefaultLinkInversion(EffectManifestParameter::LinkInversion::NotInverted);
```

Reply:
I agree, that it should be not inverted which is the default value.